components/List/ListContainer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { StyleSheet, ScrollView } from 'react-native';
+import { ListType } from '../../ambient';
+import { ListItem } from './ListItem';
+
+interface ListContainerProps {
+  lists: ListType[];
+}
+
+export const ListContainer: React.FC<ListContainerProps> = ({ lists }) => {
```

Comment:
Should we be more consistent and specify return type everywhere?

Reply:
Yup, extending the linter is on my todo list, I'll go after them all one by one in another PR.